js/tiles.js

```diff
@@ -240,11 +253,7 @@ const door = {
     },
     /* Override the function */
     visit: function() {
-      swal({
-        type: 'success',
-        title: 'You win!',
-        text: "Yay! You have won the game"
-      });
+      alert()
```

Comment:
I do not like this change as it creates an alert box which we removed previously. Why do you remove it?